livre/style.css

```diff
@@ -136,4 +141,40 @@ body > .is-root-container,
 .wp-block-post-date a:hover,
 .wp-block-site-title a:hover {
 	text-decoration: underline;
+}
+
+/*
+ * Comment form cleanup.
+ */
+
+input {
+	font-family: inherit;
+}
+
+textarea,
+input:not([type="submit"]) {
```

Comment:
Should this also be :not([type="button"]) ?

Reply:
Yes, it should. Thanks for catching that! Added in bf2ce20.